src/custom/pages/Profile/index.tsx

```diff
@@ -45,6 +46,17 @@ export default function Profile() {
   const hasOrders = useHasOrders(account)
 
   const vCowBalance = useTokenBalance(account || undefined, chainId ? V_COW[chainId] : undefined)
+  const cowBalance = useTokenBalance(account || undefined, chainId ? COW[chainId] : undefined)
+
+  const { vested, total, unvested } = useVCowData()
+
+  // TODO: remove once this is not needed anymore
+  if (isPr || isLocal) {
```

Comment:
`isLocal` shouldn't be necessary, you dont get ANY logs in local/develop?

Reply:
If I remove `isLocal` and only leave `isPr` then I don't see logs in local.

Reply:
We will modify profile page anyways in the upcoming PR related to cow swap so this will be removed